frontend/src/components/AddPage/AddForm.tsx

```diff
@@ -0,0 +1,117 @@
+import React, {ChangeEvent, useState} from 'react';
+import {Alert, Box, Button, Snackbar, TextField} from "@mui/material";
+import axios from "axios";
+import {BpmnDiagramModel} from "../../model/BpmnDiagramModel";
+import IconButton from "@mui/material/IconButton";
+import CloseIcon from "@mui/icons-material/Close";
+
+type AddFormProps = {
+    fetchDiagrams: () => void
+}
+
+function AddForm(props: AddFormProps) {
+    const [snackbarAddOpen, setSnackbarAddOpen] = useState<boolean>(false)
+    const [snackbarErrorOpen, setSnackbarErrorOpen] = useState<boolean>(false)
```

Comment:
In all diesen Fällen, wo Du einen Initialwert angibst der dem Typen in den spitzen Klammern entsprichst, kannst du diesen Typen weglassen.
```suggestion
    const [snackbarAddOpen, setSnackbarAddOpen] = useState(false)
    const [snackbarErrorOpen, setSnackbarErrorOpen] = useState(false)
```

(ist auch Geschmackssache)